Change to how to deal with the licence, based on PR #24919
Author: J. Saßmannshausen (Imperial College London/UK)
diff --git a/Tiberius-1.1.8.orig/pyproject.toml b/Tiberius-1.1.8/pyproject.toml
index d87b27c..5060778 100644
--- a/Tiberius-1.1.8.orig/pyproject.toml
+++ b/Tiberius-1.1.8/pyproject.toml
@@ -19,8 +19,7 @@ classifiers = [
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
-license = "MIT"
-license-files = ["LICEN[CS]E*"]
+license = { file = "LICENSE" }
 dependencies = [
   "rich",
   "pyyaml"
